delta.go

```diff
@@ -24,12 +24,13 @@ const (
 
 // Delta defines the pushprom message format. It represent a change on a Prometheus metric. It implements a simplistic rpc.
 type Delta struct {
-	Type   MetricType        `json:"type"`
-	Name   string            `json:"name"`
-	Help   string            `json:"help"`
-	Method string            `json:"method"`
-	Value  float64           `json:"value"`
-	Labels prometheus.Labels `json:"labels"`
+	Type    MetricType        `json:"type"`
+	Name    string            `json:"name"`
+	Help    string            `json:"help"`
+	Method  string            `json:"method"`
+	Value   float64           `json:"value"`
+	Labels  prometheus.Labels `json:"labels"`
+	Buckets *[]float64        `json:"buckets"`
```

Comment:
Is there specific reason why this one should be a pointer?

Reply:
Yes! If not specified we will use the default buckets. We need to leave that decision to the user. 
`[]float64{}` (empty slice) could have that meaning but I think it's weirder

Reply:
What if user would also provide empty slice? Should we use default buckets then as well? Also isn't it weirder to expect pointer to a pointer (slice is a pointer in fact)?

Reply:
I read the prometheus source code https://github.com/prometheus/client_golang/blob/ea348d7c20ce64c3bccdac0311e8621285d1278a/prometheus/histogram.go#L149

and 
https://github.com/prometheus/client_golang/blob/master/prometheus/histogram.go#L182

so yeah. I'll do that too to simplify. thanks!